app/Rules/SettingCaptchaQuestions.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Rules;
+
+use App\Http\Controllers\PublicWikiController;
+use Illuminate\Contracts\Validation\Rule;
+
+class SettingCaptchaQuestions implements Rule
+{
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        $value = json_decode($value, true);
+
+        if ($value === null) {
+            return false;
+        }
+
+        foreach (array_keys($value) as $question) {
+            if (!is_string($question)) {
+                return false;
+            }
+            if (strlen($question) > 200) {
+                return false;
+            }
+        }
+
+        foreach (array_values($value) as $answers) {
+            if (!is_array($answers)) {
+                return false;
+            }
+            if (count($answers) === 0) {
+                return false;
+            }
+            foreach ($answers as $answer) {
+                if (!is_string($answer)) {
+                    return false;
+                }
+                if (strlen($question) > 200) {
```

Comment:
```suggestion
                if (strlen($answer) > 200) {
```

Reply:
Thanks! I'm a fool and clearly having validation tests for this would have caught my mistake.

Reply:
I'm actually even more of a fool; apparently I'd failed to push the right amended commit